hack/testing/logging.sh

```diff
@@ -441,9 +470,30 @@ function removeAdminCert() {
   os::cmd::expect_success "oc patch secret logging-elasticsearch -p '{\"data\":{\"admin-cert\": null}}'"
 }
 
+function rebuildVersion() {
```

Comment:
for some reason the source ref is changing when i'm rebuilding the deployer for the patching test.  that is why the test is failing -- it is trying to run the deployer out of master.